### DISCUSSION TITLE
Undo styling?

### DISCUSSION THREAD

**Discussion by KennyDenvers** [HEmile/juggl#108] [Q&A]
I applied some styling to a selection of nodes while just playing around with Juggl (still very new to it). Then I wanted to return to the original styling of the nodes - and did not know how to do that. Is there a way?

(You could say, well just select the style that they had before - but just imagine I did not remember that style or also they had different "styles" (colors) before individually.)

**Comment by HEmile** [comment]
What do you mean with the original styling? The one with the default colours for tags?
You can technically completely uninstall Juggl (make sure the data.json file is deleted), and then it will regenerate this styling.

**Reply by KennyDenvers** [reply]
Have to go back into this to be able to describe more specifically. Thanks so far.
